sale_payment_sheet/models/sale_payment_sheet.py

```diff
@@ -136,8 +136,6 @@ def button_confirm_sheet(self):
                 {
                     "name": sheet.name,
                     "date": sheet.date,
-                    "journal_id": sheet.journal_id.id,
-                    "user_id": sheet.user_id.id,
```

Comment:
Why this change?

Reply:
And the salesman?

Reply:
I add journal_id again, user_id does not exist anymore in v16.
I did not realize previous PR delete this two lines.

Reply:
OK. It seems the journal is added later in this method. Any reason?